test/unit/test.amo-client.js

```diff
@@ -328,18 +328,57 @@ describe('amoClient.Client', function() {
 
     it('makes requests with an auth token', function(done) {
       var self = this;
+      var request = {url: '/somewhere'};
 
-      this.client.get({url: '/somewhere'}).then(function() {
+      this.client.get(request).then(function() {
         var call = self.client._request.calls[0];
         var headerMatch = call.conf.headers.Authorization.match(/JWT (.*)/);
         var token = headerMatch[1];
         var data = jwt.verify(token, self.client.apiSecret);
         expect(data.iss).to.be.equal(self.client.apiKey);
         expect(data).to.have.keys(['iss', 'iat', 'exp']);
+        expect(call.conf).to.be.deep.equal(
+            self.client.configureRequest(request));
         done();
       }).catch(done);
     });
 
+    it('lets you configure a request directly', function() {
+      var conf = this.client.configureRequest({url: '/path'});
+      expect(conf).to.have.keys(['headers', 'url']);
+      expect(conf.headers).to.have.keys(['Accept', 'Authorization']);
+    });
+
+    it('preserves request headers', function() {
+      var headers = {'X-Custom': 'thing'};
+      var conf = this.client.configureRequest({
+        url: '/path',
+        headers: headers,
+      });
+      expect(conf.headers['X-Custom']).to.be.equal(headers['X-Custom']);
```

Comment:
You're never going to update it so it doesn't matter. And find/replace is going to sort that out in a matter of seconds.

It's more clear. When someone looks at this they don't have to look back to verify that `headers['X-Custom']` actually has a meaningful value and there's no chance of typoing the key and having the test stop working.